src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    > 
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      value="crossorigin"
+    > 
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet"
+    >  
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <div class="card__image">
+        <img 
+          src="images/imac.jpeg" 
+          alt="imac"
+          class="image">
+      </div>
+      <div class="card__description">
+        <div class="card__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+        <div class="card__code">
+          Код товара: 195434
+        </div>
+      </div>
+      <div class="card__rating">
+        <div class="card__stars">
+          <div class="card__stars"></div>
```

Comment:
```suggestion
          <div class="stars__star"></div>
```
it is separate block